tests/engine/unit/utils.ts

```diff
@@ -20,7 +20,7 @@ export function segmentCondition() {
 }
 
 export function traitMatchingSegment() {
-    return new TraitModel(segmentCondition().property_ as string, segmentCondition().value);
+    return new TraitModel(segmentCondition().property as string, segmentCondition().value);
```

Comment:
Changed this, I didn't see the point anymore but maybe I am missing some context